packages/dx-react-scheduler-demos/src/demo-sources/scheduler-featured-appearance-customization/material-ui/demo.jsx

```diff
@@ -57,135 +55,211 @@ const getIconById = (id) => {
+  headerCellHighPriority: `${PREFIX}-headerCellHighPriority`,
+};
+// #FOLD_BLOCK
+const stylesByPriority = priorities.reduce((acc, priority) => ({
+  ...acc,
+  [`cell${priority.text.replace(' ', '')}`]: {
+    backgroundColor: alpha(priority.color[400], 0.1),
+    '&:hover': {
+      backgroundColor: alpha(priority.color[400], 0.15),
+    },
+    '&:focus': {
+      backgroundColor: alpha(priority.color[400], 0.2),
+    },
+  },
+  [`headerCell${priority.text.replace(' ', '')}`]: {
+    backgroundColor: alpha(priority.color[400], 0.1),
+    '&:hover': {
+      backgroundColor: alpha(priority.color[400], 0.1),
+    },
+    '&:focus': {
+      backgroundColor: alpha(priority.color[400], 0.1),
+    },
+  },
+}), {});
+// #FOLD_BLOCK
+const groupingStyles = ({ theme }) => ({
+  [`&.${classes.cellLowPriority}`]: stylesByPriority.cellLowPriority,
+  [`&.${classes.cellMediumPriority}`]: stylesByPriority.cellMediumPriority,
+  [`&.${classes.cellHighPriority}`]: stylesByPriority.cellHighPriority,
+  [`&.${classes.headerCellLowPriority}`]: stylesByPriority.headerCellLowPriority,
+  [`&.${classes.headerCellMediumPriority}`]: stylesByPriority.headerCellMediumPriority,
+  [`&.${classes.headerCellHighPriority}`]: stylesByPriority.headerCellHighPriority,
+  [`& .${classes.icon}`]: {
+    paddingLeft: theme.spacing(1),
+    verticalAlign: 'middle',
+  },
+});
+
+// #FOLD_BLOCK
+const StyledToolbarFlexibleSpace = styled(Toolbar.FlexibleSpace)(({ theme: { spacing } }) => ({
+  [`& .${classes.flexibleSpace}`]: {
     margin: '0 auto 0 0',
   },
-  prioritySelector: {
-    marginLeft: theme.spacing(2),
+  marginLeft: spacing(2),
+  [`& .${classes.prioritySelector}`]: {
     minWidth: 140,
     '@media (max-width: 500px)': {
       minWidth: 0,
       fontSize: '0.75rem',
-      marginLeft: theme.spacing(0.5),
+      marginLeft: spacing(0.5),
     },
   },
-});
+}));
+
 // #FOLD_BLOCK
-const usePrioritySelectorItemStyles = makeStyles(({ palette, spacing }) => ({
-  bullet: ({ color }) => ({
+const StyledPrioritySelectorItem = styled('div')(({ theme: { palette, spacing }, color }) => ({
+  [`& .${classes.bullet}`]: {
     backgroundColor: color ? color[400] : palette.divider,
     borderRadius: '50%',
     width: spacing(2),
     height: spacing(2),
     marginRight: spacing(2),
     display: 'inline-block',
-  }),
-  prioritySelectorItem: {
+  },
+  [`& .${classes.prioritySelectorItem}`]: {
     display: 'flex',
     alignItems: 'center',
   },
-  priorityText: {
+  [`& .${classes.priorityText}`]: {
     '@media (max-width: 500px)': {
       display: 'none',
     },
   },
-  priorityShortText: {
+  [`& .${classes.priorityShortText}`]: {
     '@media (min-width: 500px)': {
       display: 'none',
     },
   },
 }));
 // #FOLD_BLOCK
-const useTooltipContentStyles = makeStyles(theme => ({
-  content: {
-    padding: theme.spacing(3, 1),
+const StyledWeekViewTimeTableCell = styled(WeekView.TimeTableCell)(
+  ({ theme: { palette, spacing }, color }) => ({
```

Comment:
For that component no such classes. There are only cellLowPriority, cellMediumPriority and cellHighPriority